spec.html

```diff
@@ -39843,6 +39857,82 @@ <h1>Array.from ( _items_ [ , _mapper_ [ , _thisArg_ ] ] )</h1>
         </emu-note>
       </emu-clause>
 
+      <emu-clause id="sec-array.fromasync">
+        <h1>Array.fromAsync ( _asyncItems_ [ , _mapper_ [ , _thisArg_ ] ] )</h1>
+
+        <p>This function performs the following steps when called:</p>
+        <emu-alg>
+          1. Let _C_ be the *this* value.
+          1. If _mapper_ is *undefined*, then
+            1. Let _mapping_ be *false*.
+          1. Else,
+            1. If IsCallable(_mapper_) is *false*, throw a *TypeError* exception.
+            1. Let _mapping_ be *true*.
```

Comment:
This similarly feels simpler to me

```suggestion
          1. Let _mapping_ be *false*.
          1. If _mapper_ is not *undefined*, then
            1. If IsCallable(_mapper_) is *false*, throw a *TypeError* exception.
            1. Set _mapping_ to *true*.
```